clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -1407,14 +1407,61 @@ check_filter_endpoint()
     return true;
 }
 
+bool
+check_timestamps_increase_monotonically(void)
+{
+    // Positive test: timestamps increase monotonically.
```

Comment:
We've switched from "positive" vs "negative" to "correct" vs "incorrect" in new tests b/c there was some confusion over the former.

Reply:
I've changed postive/negative to correct/incorrect.